### PR TITLE
In BackOffice, removes unused RTL files and its calls

### LINKED ISSUE #26399: [BO] Useless file (rtl.js) should be removed
#### Describe the bug

There is a file [rtl.js](https://github.com/PrestaShop/PrestaShop/blob/develop/js/rtl.js) inside [PrestaShop/js/](https://github.com/PrestaShop/PrestaShop/tree/develop/js) folder. This file tries to fix all inline styles in a page using jQuery for RTL by searching inline styles like `text-align`, `float` and `background-position` and changing their values from `left` to `right`. 

I think it was useful for old theme and in new theme there is no inline styles. 

Old theme (Inline style)
`<div id="sample" style="float:left"></div>`

New theme (Clean CSS)
`<div id="sample" class="float-left"></div>`

#### Expected behavior

The `rtl.js` and calls to this file should be removed.

#### Steps to Reproduce

I removed this file and there was no problem for RTL mode.

#### Additional information

* PrestaShop version: 1.7.8.0
* PHP version: 7.4

### PR DESCRIPTION
**Pull request by mparvazi**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | The `rtl.js` file is too old and executing its script in BO pages won't have any effect.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | Fixes #26399
| How to test?      | Currently running this script hasn't any effect on display of BO. This file is visible in the browser console if the employee's language is RTL.
| Possible impacts? | -

Screenshot 2021-11-15 182639

# BC Break
This file is used in order to do some RTL stuff inside inline styles, maybe some modules are using it, so they will need to fix it (anyway it's not a good practice, so we accept this BC break without any problems!)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26616)

### CODE REVIEW

File: js/rtl.js [hunk 1]
```diff
@@ -1,50 +0,0 @@
-/*
- * dmartl.js
- * DM Auto RTL - Auto RTL all inline style in page using jquery
- * Autor: Danoosh Miralayi
- * Website: presta-shop.ir
- * License: MIT
- * Find it here: https://github.com/Danoosh/DM-Auto-RTL
```
Comment (matks) [inline js/rtl.js:7]: Maybe we can ask @Danoosh if he remembers why this file was introduced, and tell us if it is still relevant ? :)

Comment (mparvazi) [reply js/rtl.js:7]: I think it was useful for early versions of 1.6.

Comment (mparvazi) [reply js/rtl.js:7]: In PS 1.7, I couldn't find any inline styles that requires compatibility for RTL languages (e.g. text-align, float or background-position).

**Review by tegbessou — APPROVED** [review]
Thank you for your PR!

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
@mparvazi I approve this PR, it looks like we don't use it at all. I tagged it as a BC break because some modules can benefit of this files.

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
I guess there is no test needed ?

**Comment by mparvazi** [comment]
Hello @matthieu-rolland 

This code runs on all BO pages without changing the page elements, because there is nothing to change!

It's just a  jquery search for dom elemets, not a function or library that can be called in legacy codes. I think deleting it won't cause an execution error. 

It's clear that this code has no effect on new-theme, I tested on legacy theme pages (Dashboard, Stats, ...) and couldn't find any errors in display.

**Comment by NeOMakinG** [comment]
@PrestaShop/qa-team This file is useless, do we need to test this PR?

**Comment by Robin-Fischer-PS** [comment]
Hi @NeOMakinG ! Are we sure it's not used in any Legacy pages ?

**Review by Progi1984 — APPROVED** [review]

**Comment by NeOMakinG** [comment]
> Hi @NeOMakinG ! Are we sure it's not used in any Legacy pages ?

I guess it's a very old "unused by the core" file we should get rid of because we've better solutions in order to manage the RTL part, it's not probably used by the core even on legacy page atm but maybe it's used inside some modules.

But I think that module have access to the RTL PHP variable, this means that they are able to include an RTL stylesheets by their own without the need of this, we should remove this file to clean the project and every others external inputs needs to be changed if we notice some

**Comment by Robin-Fischer-PS** [comment]
So we will make a quick test with RTL on the BO (and some native modules), 15mn max, before validating the PR !

**Comment by khouloudbelguith** [comment]
Hi @mparvazi,

It is ok :heavy_check_mark: 
https://watch.screencastify.com/v/qQMNYFth3T50Y0IbS803
**With PR**
image
**Without Pr**
image

Thanks!

**Comment by Progi1984** [comment]
Thanks @mparvazi & @khouloudbelguith

**Pull request merged**